src/powershell/enable-rdp.ps1

```diff
@@ -28,3 +28,26 @@ if ($(Get-NetFirewallRule -Name $name).Enabled -ne $true) {
     Write-Error "[X] Failed to verify firewall rule: $name" -ErrorAction Stop
 }
 Write-Output "[*] Firewall rule successfully verified: $name"
+
+# Allow Terminal Server shadowing through the Windows firewall
+$name = "Remote Desktop - Shadow (TCP-In)"
+Write-Output "[ ] Configuring firewall rule: $name"
+Set-NetFirewallRule -DisplayName $name -Enabled True -Profile Private
+if ($(Get-NetFirewallRule -DisplayName $name).Enabled -ne $true) {
+    Write-Error "[X] Failed to verify firewall rule: $name" -ErrorAction Stop
+}
+Write-Output "[*] Firewall rule successfully verified: $name"
+
+# Enable Terminal Server shadowing
+$name = "Shadow"
+$path = 'HKLM:\SYSTEM\CurrentControlSet\Control\Terminal Server\WinStations\RDP-Tcp'
+Write-Output "[ ] Configuring setting: $name"
+Set-ItemProperty -Path $path -Name $name -Value "1"
+if ($(Get-ItemProperty -Path $path -Name $name).Shadow -ne 1) {
+    Write-Error "[X] Failed to verify setting: $name" -ErrorAction Stop
+}
+Write-Output "[*] Setting successfully verified: $name"
+
+# Give administrator account full control of Terminal Server
+Get-CimInstance -Namespace root\CIMV2\TerminalServices -ClassName Win32_TSPermissionsSetting -Filter 'TerminalName ="RDP-Tcp"' |
+        Invoke-CimMethod -MethodName AddAccount -Arguments @{AccountName="BUILTIN\Administrators"; PermissionPreSet="2"}
```

Comment:
So does this mean that the `administrator` user account is the only one that is allowed to do RDP shadowing?

Reply:
yes, that is correct. it is also possible to grant permission to specific groups. there is more info regarding this at the bottom of this page http://woshub.com/rds-shadow-how-to-connect-to-a-user-session-in-windows-server-2012-r2/#h2_5

Reply:
I'm not sure if our end users have a need for other users besides `administrator` do shadowing.  For now, this is fine, but can you please add a comment making this limitation clear?

Reply:
It is my understanding that `BUILTIN\Administrators` is a group and different from the built-in `Administrator` account. The `Administrator` account is part of the `BUILTIN\Administrators` group but multiple accounts can be in that group.

Reply:
@dav3r @mcdonnnj is this conversation related to the conversation below that I have resolved?

Reply:
I think we are ok here since [the comment now specifies](https://github.com/cisagov/windows-server-packer/pull/22/commits/1a112656348412dc91b855a66160f9c26aeffded) the `Administrators` group, rather than a single `Administrator` account.  @mcdonnnj, please confirm.